crate/content.md

```diff
@@ -1,90 +1,113 @@
 
-If you want to attach a persistent data directory, add the following parameter.
+To form a cluster from scratch, start a few instances of the Crate container as a background daemon:
 
 ```console
-$ docker run -d -p 4200:4200 -p 4300:4300 -v <data-dir>:/data crate
+# docker run -d crate crate
 ```
 
-### Use custom Crate configuration
-
-Use the following parameter to provide your custom [Crate configuration file](https://crate.io/docs/reference/configuration.html).
+To access the admin UI, map port 4200 and point your browser to port tcp/4200 of a node of your choice while you start it or look up its IP later:
 
 ```console
-$ docker run -d -p 4200:4200 -p 4300:4300 crate -Des.config=/path/to/crate.yml
+# firefox "http://$(docker inspect --format='{{.NetworkSettings.IPAddress}}' $(docker run -d crate crate)):4200/admin"
 ```
 
-Any configuration settings may be specified upon startup using the `-D` option prefix. For example, configuring the cluster name by using system properties will work this way:
+For production use it's strongly recommended to use only one container per machine. This will give the best possible performance and by mapping the ports from the Docker container to the host it acts like a native installation. Crate's default ports 4200 (HTTP) and 4300 (Transport protocol).
 
 ```console
-$ docker run -d -p 4200:4200 -p 4300:4300 crate crate -Des.cluster.name=cluster
+# docker run -d -p 4200:4200 -p 4300:4300 crate crate
 ```
 
-### Environment
+## Attach Persistent Data Directory
+
+Crate stores all important data in */data*. It's advised to mount this directory to avoid writing within the docker image:
 
-To set environment variables for Crate Data you need to use the `--env` option when starting the docker image.
+```console
+# docker run -d -v <data-dir>:/data crate crate
+```
+
+## Use Custom Crate Configuration
 
-For example, setting the heap size:
+Crate is controlled by a single configuration file which has sensible defaults already. If you derive your container from the Crate container, make sure to place your file inside it and let Crate know where to find it:
 
 ```console
-$ docker run -d -p 4200:4200 -p 4300:4300 --env CRATE_HEAP_SIZE=32g crate
+# docker run -d crate crate -Des.config=</path/to>/crate.yml
 ```
 
-## Multicast
+Other configuration settings may be specified upon startup using the `-D` option prefix. For example, configuring the cluster name by using system properties works like this:
 
-Crate uses multicast for node discovery by default. However, Docker does only support multicast on the same host. This means that nodes that are started on the same host will discover each other automatically, but nodes that are started on different hosts need unicast enabled.
+```console
+# docker run -d crate crate -Des.cluster.name=<my-cluster-name>
+```
+
+For further configuration options refer to the[Configuration](https://crate.io/docs/stable/configuration.html) section of our documentation.
```

Comment:
Another case of `the` tightly coupled with `[` ("refer to the[Configuration](https://crate.io/docs/stable/configuration.html) section").